admin_ui/app/role/RoleStore.js

```diff
@@ -0,0 +1,15 @@
+import { BehaviorSubject } from 'rxjs';
+
+class RoleStore {
+  constructor() {
+    // The deleteModal$ observable takes 3 possible values:
+    // "" (empty string), "DELETE", "DELETION_IN_PROGRESS"
+    this.deleteModal$ = new BehaviorSubject('');
+    // Useful flag to indicate user has changed the roles organization
+    this.changedOrg$ = new BehaviorSubject(false);
```

Comment:
Let's discuss in person how this stream is actually used - there might be a better way to do (or maybe not).